packages/SwingSet/test/abandon-export.test.js

```diff
@@ -180,17 +205,11 @@ async function doAbandon(t, reachable) {
     await flushDeliveriesA();
     // vatB should not get a dispatch.dropImports
     t.deepEqual(logB, []);
-    // the object still exists, now only recognizable
-    targetRefCount = kvStore.get(`${targetKref}.refCount`);
-    expectedRefCount = '0,1';
-    t.is(targetRefCount, expectedRefCount); // merely recognizable
+    // the kernel automatically retires the orphaned
+    // now-merely-recognizable object
+    t.deepEqual(logA, [{ type: 'retireImports', vrefs: [targetForAlice] }]);
```

Comment:
Internal consistency nit:
```suggestion
    assertSingleEntryLog(t, logA, 'retireImports', { vrefs: [targetForAlice] });
```